README.md

```diff
@@ -90,7 +90,8 @@ Type: `String[]`<br>
 Default: `[]`
 
 ### require
-Require files/dir before executing features. If you apply a path don't use globbing as it is not supported by Cucumber. Instead just reference the directory where you step definitions are located (e.g. `/path/to/definitions` instead of `/path/to/definitions/*`).
+**Please note that globbing is not supported by Cucumber, paths must adhere to Node module resolution**
+Require files before executing features (e.g. `/path/to/module` or `/path/to/module-directory` where directory contains `index.js`).
```

Comment:
But it's not due to Node 8, it's the new Cucumber release bump.

If I use Node 8 with `wdio-cucumber-framework@0.3.1` everything works great, the old whole-directory include `require` behaviour is present.

After upgrading to `wdio-cucumber-framework@1.0.0`, I need to update my `require` config to use Node module resolution, no more whole-directory includes.